x/bridge/keeper/msg_server.go

```diff
@@ -28,7 +28,7 @@ func (m msgServer) InboundTransfer(
 ) (*types.MsgInboundTransferResponse, error) {
 	err := msg.ValidateBasic()
 	if err != nil {
-		return nil, err
+		return nil, errorsmod.Wrapf(sdkerrors.ErrInvalidRequest, err.Error())
```

Comment:
Was there a reason for this? If you are just following the behavior somewhere else in the code that is fine, I just imagine using the error returned is probably fine though